lib/ansible/utils/display.py

```diff
@@ -225,6 +225,8 @@ def __init__(self, verbosity=0):
             try:
                 cmd = subprocess.Popen([self.b_cowsay, "-l"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
                 (out, err) = cmd.communicate()
+                if cmd.returncode:
+                    raise Exception
```

Comment:
how about adding a message and a reason for the exception?

Reply:
Sure. Is there a default way/style to do so in ansible.

Reply:
You may want to use `AnsibleError`